### ISSUE TITLE
Doggo

### ISSUE THREAD

**Issue by valoeghese** [minecraft-cursed-legacy/Plasma#55]
Doggo

**Comment by hYdos** [comment]
if this happens it give me more reason to match down to b1.7.3

**Comment by hYdos** [comment]
:tiny

**Comment by valoeghese** [comment]
You can probably use https://github.com/calmilamsy/BIN-Mappings if you don't want doggo

**Comment by mineLdiver** [comment]
> You can probably use https://github.com/calmilamsy/BIN-Mappings if you don't want doggo

pull BIN when :tiny

**Comment by valoeghese** [comment]
No.

Problems with BIN:

![image](https://user-images.githubusercontent.com/46117728/109127770-a8d03200-77b3-11eb-86d8-3d25991daf9d.png)
![image](https://user-images.githubusercontent.com/46117728/109127867-c43b3d00-77b3-11eb-9889-d85c3775e8fb.png)

- And I can't find the message that says it but I think either you or calim admitted to sometimes referencing voldemap for what a method does...

**Comment by calmilamsy** [comment]
I don't think we have followed that one for a while actually. I completely forgot that rule existed, and it wouldnt be too hard to remove any `I`s from any existing classes.
And yea, I used voldemap to figure out what the hell the entitytracker/entity networking stuff does, cause good luck reading any of that stuff.
But yea, BIN will keep being its own thing so long as names like this suggestion exist in Plasma.
